fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -1718,6 +1745,194 @@ private LogicalPlan plan(ParserRuleContext tree) {
         return (LogicalPlan) tree.accept(this);
     }
 
+    /* ********************************************************************************************
+     * create table parsing
+     * ******************************************************************************************** */
+
+    @Override
+    public LogicalPlan visitCreateTable(CreateTableContext ctx) {
+        String dbName = null;
+        String tableName;
+        List<String> nameParts = visitMultipartIdentifier(ctx.name);
+        if (nameParts.size() == 1) {
+            tableName = nameParts.get(0);
+        } else if (nameParts.size() == 2) {
+            dbName = nameParts.get(0);
+            tableName = nameParts.get(1);
+        } else {
+            throw new AnalysisException("nameParts in create table should be 1 or 2");
+        }
+        KeysType keysType = null;
+        if (ctx.DUPLICATE() != null) {
+            keysType = KeysType.DUP_KEYS;
+        } else if (ctx.AGGREGATE() != null) {
+            keysType = KeysType.AGG_KEYS;
+        } else if (ctx.UNIQUE() != null) {
+            keysType = KeysType.UNIQUE_KEYS;
+        }
+        String engineName = ctx.engine != null ? ctx.engine.getText().toLowerCase() : "olap";
+        DistributionDescriptor desc = new DistributionDescriptor(ctx.HASH() != null, ctx.AUTO() != null,
+                Integer.parseInt(ctx.number().getText()),
```

Comment:
if AUTO bucket, i think ctx.number() will return null, and this statement will throw NPE.